Q1script.txt

```diff
@@ -0,0 +1,7 @@
+#SCRIPT DONE!
```

Comment:
1.	I’m taking .1 off here but this applies to the assignment as a whole. The prompt asks for a single shell script that answers all 3 questions and you’ve provided a different script for each answer
a.	0.9
2.	1
3.	Code works but gives the wrong answer. The difference in earning should be ~$4. The piece of your code that is incorrect is the call to sort. The way you have the flags formatted sort doesn’t read them quite right and so doesn’t sort the data correctly sort –n –r –k2,2 would work 
a.	0.7

2.6/3